ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.js

```diff
@@ -42,11 +40,14 @@ class Menu extends React.Component {
   onHover(active) {
     this.setState({ active })
   }
+  getChildren() {
+    return this.getChildrenFrom(this.props.children)
+  }
+  getChildrenFrom(children: any) {
+    return React.Children.toArray(children).filter(o => o)
```

Comment:
❓ What is the `.filter(o -> o)` doing?

Reply:
That's removing all nulls (and undefined but there shouldn't be any) from the array.